### PR TITLE
feat: init vuepress

### PR DESCRIPTION
**Pull request by qsvtr**
I only deleted the unused images, there will surely be components to remove during the construction of the web site

### CODE REVIEW

**Comment by dlecan** [comment]
Fixes #1

**Comment by rigwild** [comment]
You should use `computed` properties instead of `methods`.

See:
- https://vuejs.org/v2/guide/computed.html#Computed-Caching-vs-Methods).
- https://vuejs.org/v2/guide/computed.html#Basic-Example

**Comment by dlecan** [comment]
@rigwild PR welcome :smile: 
We won't do anything as we don't see any performance issues with existing pages.

**Comment by dlecan** [comment]
@qsvtr
Netlify was misconfigured so it didn't trigger a deployment website preview.

As your commit message doesn't follow [conventional commit](https://www.conventionalcommits.org/) and it is mandatory in this repo, can you update it and force-push?

This should trigger a new Netlify build.

File: docs/.vuepress/components/Footer.vue [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<template>
+  <footer>
+    © 2021 <a href="https://www.unik-name.com">Unikname</a>&nbsp;&bullet;&nbsp;<a href="/powered-by-ark-io">Powered by ARK</a>
```
Comment (dlecan) [inline docs/.vuepress/components/Footer.vue:3]:
```suggestion
    © 2021 <a href="https://www.unik-name.com">Unikname</a>&nbsp;&bullet;&nbsp;<a href="https://docs.uns.network/powered-by-ark-io">Powered by ARK</a>
```

**Comment by dlecan** [comment]
@qsvtr one last thing: a link is broken :)

**Review by dlecan — APPROVED** [review]

**Comment by dlecan** [comment]
Thank you!

**Pull request merged**